auction_keeper/main.py

```diff
@@ -298,7 +298,10 @@ def approve(self):
         self.strategy.approve(gas_price=self.gas_price)
         time.sleep(2)
         if self.dai_join:
-            self.mcd.approve_dai(usr=self.our_address, gas_price=self.gas_price)
+            if self.mcd.dai.allowance_of(self.our_address, self.dai_join.address) > Wad.from_number(2**50):
```

Comment:
That's 100 trillion, which is orders of magnitude higher than the the amount of Dai in existence, let alone how much Dai a single actor might ever want to `join`.

Reply:
Lol - Right- but it checks to see whether the standard approval has been done.